src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { handleCheckboxChange } from "../../utils/helper";
+import CheckboxWrapper from "./checkbox";
+
+function Checkbox({
+  employeeId,
+  deleteCheckBoxesList,
+  employeesIdList,
+}: {
+  employeeId?: string;
+  deleteCheckBoxesList: {
+    checkedBoxesList: string[];
+    setCheckedBoxesList: React.Dispatch<React.SetStateAction<string[]>>;
+  };
+  employeesIdList?: string[];
+}) {
+  return (
+    <CheckboxWrapper
+      className="checkbox"
+      type="checkbox"
+      onClick={(e) => e.stopPropagation()}
+      onChange={() =>
+        handleCheckboxChange({
+          employeeId,
+          deleteCheckBoxesList,
+          employeesIdList,
+        })
+      }
+      checked={
+        employeeId
+          ? deleteCheckBoxesList.checkedBoxesList.includes(employeeId)
```

Comment:
these much conditions needed? if so , please get the state to a const and use it here

Reply:
Resolved in 70b4934300f70439e5aca8f8c21bc6affc0a87e5